kolena/_experimental/instance_segmentation/dataset.py

```diff
@@ -11,26 +11,54 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from typing import Dict
+from typing import Literal
+from typing import Union
+
+import pandas as pd
+
 from kolena._experimental.object_detection.dataset import upload_object_detection_results
 
 
-upload_instance_segmentation_results = upload_object_detection_results
-"""
-Compute metrics and upload results of the model for the dataset.
+def upload_instance_segmentation_results(
+    dataset_name: str,
+    model_name: str,
+    df: pd.DataFrame,
+    *,
+    ground_truths_field: str = "ground_truths",
+    raw_inferences_field: str = "raw_inferences",
+    iou_threshold: float = 0.5,
+    threshold_strategy: Union[Literal["F1-Optimal"], float, Dict[str, float]] = "F1-Optimal",
+    min_confidence_score: float = 0.01,
+    batch_size: int = 10_000,
+) -> None:
+    """
+    Compute metrics and upload results of the model for the dataset.
 
-Dataframe `df` should include a `locator` column that would match to that of corresponding datapoint. Column
-:inference in the Dataframe `df` should be a list of scored [`Polygons`][kolena.annotation.Polygon].
+    Dataframe `df` should include a `locator` column that would match to that of corresponding datapoint. Column
+    :inference in the Dataframe `df` should be a list of scored [`Polygons`][kolena.annotation.Polygon].
```

Comment:
Not new to this change, but it looks like the formatting for `:inference` isn't working as intended -- not sure if there's a quick fix or not.

Reply:
Good catch. I updated the docstring to match how `locator` column is referenced prior